compose-table/src/androidTest/java/org/dhis2/composetable/TableRobot.kt

```diff
@@ -0,0 +1,76 @@
+package org.dhis2.composetable
+
+import androidx.annotation.DrawableRes
+import androidx.compose.ui.test.SemanticsMatcher
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.ComposeContentTestRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.performImeAction
+import androidx.compose.ui.test.performTextClearance
+import androidx.compose.ui.test.performTextInput
+import org.dhis2.composetable.ui.CELL_TEST_TAG
+import org.dhis2.composetable.ui.DrawableId
+import org.dhis2.composetable.ui.INPUT_TEST_FIELD_TEST_TAG
+import org.dhis2.composetable.ui.INPUT_TEST_TAG
+import org.dhis2.composetable.ui.ROW_TEST_TAG
+
+fun tableRobot(
+    composeTestRule: ComposeContentTestRule,
+    tableRobot: TableRobot.() -> Unit
+) {
+    TableRobot(composeTestRule).apply {
+        tableRobot()
+    }
+}
+
+class TableRobot(
+    private val composeTestRule: ComposeContentTestRule,
+) {
```

Comment:
👏🏻👏🏻 👏🏻 👏🏻 👏🏻 👏🏻 
Do you think this robot could get very big? Do you think it makes sense to name it CellTableRobot?

Reply:
We'll see. I think we can rename it and create other robots when needed.